games-util/gamemode/gamemode-1.5.1.ebuild

```diff
@@ -64,14 +68,17 @@ pkg_pretend() {
 }
 
 multilib_src_configure() {
-	local myconf=()
+	local emesonargs=(
+		-Dwith-systemd=true
```

Comment:
I think this makes it search for libsystemd. You should try to make it a conditional dep.

```
Run-time dependency libsystemd found: NO (tried pkgconfig and cmake)

meson.build:83:0: ERROR: Dependency "libsystemd" not found, tried pkgconfig and cmake

A full log can be found at /tmp/portage/games-util/gamemode-1.5.1/work/gamemode-1.5.1-abi_x86_32.x86/meson-logs/meson-log.txt
 * ERROR: games-util/gamemode-1.5.1::testworld failed (configure phase):
 *   (no error message)
```

Reply:
Yeah, let's make a useflag then. We are also linking to systemd in one of the libs, so switching systemd should rebuild this package.